helion/_compat.py

```diff
@@ -253,7 +253,15 @@ def _min_dot_size(
 
     if torch.xpu.is_available():
         # pyrefly: ignore [missing-import]
-        from triton.backends.intel.compiler import min_dot_size as min_dot_size_xpu
+        try:
+            # Support New triton API which put the min_dot_size in XPUBackend class
+            from triton.backends.intel.compiler import XPUBackend
```

Comment:
CUDA does not do it. I'm wondering why we need to do it. @etiotto, may I know the background?